src/controllers/queries/__tests__/index.test.ts

```diff
@@ -379,7 +375,7 @@ describe('QueriesController', () => {
             id: 'ccaaaaaa-0000-4000-8000-d8ae0805059e',
             type: 'https://github.com/digicatapult/veritable-documentation/tree/main/schemas/veritable_messaging/query_types/total_carbon_embodiment/request/0.1',
             createdTime: 1,
-            expiresTime: 1,
+            expiresTime: Math.floor(expiresAt.getTime() / 1000),
```

Comment:
could you use `expiresAt.toLocaleTimeString()`?

Reply:
Or maybe we leave as is and include this in ticket https://digicatapult.atlassian.net/browse/VR-418